Jenkinsfile

```diff
@@ -3,6 +3,24 @@
 fhBuildNode {
     stage('Install Dependencies') {
         npmInstall {}
+        withPrivateNPMRegistry {
+            sh 'npm install fh-dfc@0.24.4-75'
+        }
+    }
+
+    stage('Lint') {
+        print 'Lint task is broken see https://issues.jboss.org/browse/FH-2716'
+        //sh 'grunt jshint'
```

Comment:
The issue linked here above is the wrong one (fh-mbaas-express vs. fh-mbaas). The correct one is https://issues.jboss.org/browse/FH-3611 (I've updated it in the PR description).

Also, while changing that, consider changing the commented-out line to be `//sh 'grunt eslint'`, since that's what it will be when it works.